plone/app/layout/viewlets/content.py

```diff
@@ -71,7 +71,7 @@ def show(self):
             ISecuritySchema,
             prefix='plone',
         )
-        return not self.anonymous or settings.allow_anon_views_about
+        return self.anonymous and settings.allow_anon_views_about
```

Comment:
It'd be nice to add a test here so we don't get a regression in this important line again.